hexcore/infrastructure/api/app.py

```
@@ -153,6 +191,65 @@ def create_app(
+    Las excepciones de los plugins de Darwin, si el módulo está cableado.
+
+    Se consulta el contenedor y no un registro global: los plugins son de un despliegue, no
+    del proceso. Si Darwin todavía no se configuró —`create_app` antes de
+    `configure_identity`— devuelve vacío en vez de lanzar: el orden de cableado es del
+    consumidor, y hacer que el orden importe convertiría un detalle en un error de arranque.
+    """
+    from hexcore.darwin.application.container import get_identity_container
+
+    try:
+        contenedor = get_identity_container()
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)